core/message-tags-3.3.md

```diff
@@ -80,6 +89,24 @@ Individual tag keys MUST only be used a maximum of once per message. Clients
 receiving messages with more than one occurrence of a tag key SHOULD discard all
 but the final occurrence.
 
+### Empty tagged messages
+
+Servers MUST accept `PRIVMSG` and `NOTICE` events sent with client-only tags even if
```

Comment:
> This can't be called for PRIVMSG without text as it would be indistinguishable from PRIVMSG with empty text.

What's the consequence of losing this distinction? Is there a practical issue with "casting" a missing param to an empty param before calling the callback?